app/services/update/update_client_service.py

```diff
@@ -224,7 +224,9 @@ def update_resource(
                         continue
                     targets.append(e)
 
-            self.__clear_and_add_nodes(self.update_page(targets, adjacency_map_service))
+            nodes = self.update_page(targets, adjacency_map_service, directory.endpoint)
```

Comment:
See below. This can then also be removed
```suggestion
            nodes = self.update_page(targets, adjacency_map_service)
```